network.js

```diff
@@ -32,6 +32,6 @@ export default {
       ],
     },
   },
-  icon: '~/assets/images/currencies/atom.png',
-  localSigning: true,
+  icon: `https://lunie.fra1.digitaloceanspaces.com/network-icons/cosmos.png`,
```

Comment:
why? 

we will not keep this image on DO...

Reply:
true, this is a bad idea

Reply:
nuxt is handling the dynamic assets not well so this was a workaround

Reply:
what is the problem with nuxt / where did we see this? not a big deal just curious.